release-checks.sh

```diff
@@ -50,6 +50,13 @@ then
     exit 1
 fi
 
+ORIGIN_URL=$(git remote get-url --push origin)
+if [[ $(echo ${ORIGIN_URL} | grep -c "elastic/rally") == "0" ]]
```

Comment:
How about something simpler here like `if [[ $ORIGIN_URL != *"elastic/rally"* ]]`?

Reply:
Thanks for the suggestion. I've pushed this change in https://github.com/elastic/rally/pull/762/commits/ae88a21d00435f61fcaca162436a467966cea4e5.